config/locales/client.en.yml

```diff
@@ -16,8 +16,8 @@ en:
     date_range:
       today: 'Today'
       yesterday: 'Yesterday'
-      this_week: 'This Week'
-      this_month: 'This Month'
+      thisweek: 'This Week'
+      thismonth: 'This Month'
```

Comment:
is there a risk that this is introducing another problem rather than solving anything?